File: app/src/main/java/com/example/admin/logoguess/info.java
package com.example.admin.logoguess;

import android.app.Activity;
import android.content.Intent;
import android.os.Bundle;
import android.view.View;

public class info extends Activity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.info);
    }

    public void backhome(View view) {
        Intent openStartingPoint = new Intent(this, MainActivity.class);
        startActivity(openStartingPoint);
    }

    public void onBackPressed() {


    }

}

File: app/src/main/java/com/example/admin/logoguess/info.kt
package com.example.admin.logoguess

import android.app.Activity
import android.content.Intent
import android.os.Bundle
import android.view.View

class info : Activity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.info)
    }

    fun backhome(view: View?) {
        val openStartingPoint = Intent(this, MainActivity::class.java)
        startActivity(openStartingPoint)
    }

    override fun onBackPressed() {}
}